test/simulator/timestamp/src/timestamp_manager.cpp

```diff
@@ -222,3 +222,70 @@ timestamp_manager::validate_read_timestamp(session_simulator *session, const uin
 
     return (0);
 }
+
+/*
+ * Validate the commit timestamp. The constraints on the commit timestamp are:
```

Comment:
Missing comment

Reply:
Ah I missed this. I was going to add the constraints similar to how the other validate methods do above. However, it is largely just repeating the comments within the function so it's probably not necessary.

Reply:
Yes, those in the function are already very good. Let's keep it short for the summary.